charts/hivemq-platform/DEVELOPMENT.md

```diff
@@ -1,20 +1,28 @@
 ## Development
 
-Generate templates with values to check
+Generate templates with custom values
 
 ```bash
-helm template ./hivemq-platform-operator
-helm template ./hivemq-platform
+helm template operator ./charts/hivemq-platform-operator -f my-operator-values.yaml
+helm template platform ./charts/hivemq-platform -f my-platform-values.yaml
 ```
 
-Simulate without deploying into the k8s cluster to check the correctness of the templates.
+Simulate without deploying into the K8s cluster to check the correctness of the templates.
 
-NOTE: CRD has to be installed first in the k8s cluster
+**NOTE**: CRD has to be installed first in the K8s cluster
 
 ```bash
 kind create cluster
-helm install --dry-run my-operator-release hivemq-platform-operator
-helm install --dry-run my-platform-release hivemq-platform 
+helm install operator ./charts/hivemq-platform-operator --dry-run
+helm install platform ./charts/hivemq-platform --dry-run 
+```
+
+### Unit tests
+For running unit tests, the Helm [helm-unittest](https://github.com/helm-unittest/helm-unittest?tab=readme-ov-file#helm-unittest) plugin is required to be installed as part of your Helm installation.
+
+```bash
+helm unittest ./charts/hivemq-platform-operator -f './tests/**/*_test.yaml'
```

Comment:
I've added a Gradle task `test` to execute all Helm unit tests, so you can use `./gradlew test` instead. That one also executes the legacy operator and Swarm tests though. And it doesn't use the `-f` parameter, so we have to check that it executes the same.

Beside the missing colored output ([no solution for that available yet](https://discuss.gradle.org/t/colour-output-in-console-from-gradle-exec-task/30498)), I think this might be easier to use.

Reply:
Thanks! I didn't remember we had such Gradle task. 
Indeed, we need to modify that, otherwise the tests placed in the inner folder won't be executed (and no error will be displayed either). 
At the moment it's not needed in all of our charts, but I'd prefer to keep them aligned all together with the same command.
Also updated this `DEVELOPMENT.md` files.